be/src/exec/vectorized/jdbc_scanner.cpp

```diff
@@ -439,21 +442,29 @@ Status JDBCScanner::_fill_chunk(jobject jchunk, ChunkPtr* chunk) {
         } else if (column_class == "java.sql.Timestamp") {
             DCHECK(slot_desc->type().type == TYPE_DATETIME);
             for (int i = 0; i < num_rows; i++) {
-                RETURN_IF_ERROR(_append_datetime_val(helper.list_get(jcolumn, i), slot_desc, column.get()));
+                jobject jelement = helper.list_get(jcolumn, i);
+                DeferOp defer([&jelement, this]() { _jni_env->DeleteLocalRef(jelement); });
```

Comment:
it seems jelement will be delete in _append_datetime_val 
PTAL @silverbullet233

Reply:
> it seems jelement will be delete in _append_datetime_val PTAL @silverbullet233

yes, I think this defer is no need